src/ducks/apps/index.js

```diff
@@ -251,6 +251,10 @@ function _sanitizeManifest(app) {
     !!sanitized.terms.url &&
     !!sanitized.terms.version
   if (sanitized.terms && !hasValidTerms) delete sanitized.terms
+  // remove incomplete or empty partnership
+  const hasValidPartnership =
+    !!sanitized.partnership && !!sanitized.partnership.description
+  if (sanitized.partnership && !hasValidPartnership) delete sanitized.terms
```

Comment:
Maybe we should have helpers like `parternship.isValid` and `terms.isValid` ? Those would be testable ;)

Reply:
I see no updated tests from this change :/